test/support/factories.ex

```diff
@@ -128,4 +128,12 @@ defmodule CodeCorps.Factories do
       skill: build(:skill)
     }
   end
+
+  def preview_factory do
+    %CodeCorps.Preview{
+      body: "Foo",
+      markdown: "Bar",
```

Comment:
For this part I would make the markdown and body match up. Also, does the factory need the body of it has the markdown? There seem to be inconsistencies with other factories.